scrapers/TripForFuck.yml

```diff
@@ -0,0 +1,34 @@
+name: TripForFuck
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - tripforfuck.com/member/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h1[@class="h3 mb-0"]/text()
+      Date:
+        selector: //div[@class="d-flex small movie-status"]/p
+        postProcess:
+          - replace:
+              - regex: (\d+).+
+                with: $1
```

Comment:
just a minor issue here
for old scenes eg https://www.tripforfuck.com/member/movie/001/index.html where days have more than 3 digits there is a , as a separator

something like 
```yaml
          - replace:
              - regex: (\d),(\d)
                with: $1$2
              - regex: (\d+).+
                with: $1
```
or 
```yaml
          - replace:
              - regex: ","
                with: ""
              - regex: (\d+).+
                with: $1
```
should fix the issue

Reply:
Did not notice that, Yup that fix's it for 3 digits.